articles/Third Party Tools and Libraries/enable-math.md

```diff
@@ -1,5 +1,5 @@
 <!--
-parent: Wiki
+parent: articles
```

Comment:
Parent is the category under which an article is listed e.g. on https://hub.taocloud.org/articles (Administrator Guide, Data Abstractions etc.). If set at all it should be in proper spelling, not lower case etc. On the other hand you can ommit it. In this case the directory name will be used.
Example:
taohub-articles/articles/Some Category/my-article.md (assumed the title is My Article)
will be displayed as
**Some Category**
  - My Article